<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>457</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_funcPrivPiIsOk">
      <property name="text">
       <string>privPi</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButton_funcAngle1to2IsOk">
      <property name="text">
       <string>angle_1to2</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton_funcDist1to2IsOk">
      <property name="text">
       <string>dist_1to2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_funcRadIsOk">
      <property name="text">
       <string>rad</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_funcCoordYIsOk">
      <property name="text">
       <string>coordY</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QPushButton" name="pushButton_funcNormalDistribution">
      <property name="text">
       <string>normalDistribution</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_connectionLib">
      <property name="text">
       <string>myMath.dll</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_funcGradIsOk">
      <property name="text">
       <string>grad</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_funcSignIsOk">
      <property name="text">
       <string>sign</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton_funcCoordXIsOk">
      <property name="text">
       <string>coordX</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_funcSqrIsOk">
      <property name="text">
       <string>sqr</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_funcPriv2PiIsOk">
      <property name="text">
       <string>priv2Pi</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QPushButton" name="pushButton_variance">
      <property name="text">
       <string>getVariance</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_funcTrajPartIsOk">
      <property name="text">
       <string>функция расчета участка траектории (findPrj)</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_funcTimeIsOk">
      <property name="text">
       <string>функция расчета времени на маневр (findTimeToAlterate)</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QPushButton" name="pushButton_mean">
      <property name="text">
       <string>getMean</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_funcValidateIsOk">
      <property name="text">
       <string>функция валидации данных (verifyParams)</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QPushButton" name="pushButton_funcUniformDistribution">
      <property name="text">
       <string>uniformDistribution</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_funcFindPos_sc">
      <property name="text">
       <string>модель прямолинейного равномерного движения (findPos_sc)</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_funcFindPosComplex">
      <property name="text">
       <string>модель сложного движения объекта (findPos_manHS)</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_func2Z">
      <property name="text">
       <string>метод двух замеров (estimateMotionParams_2z)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>363</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
